oex_plugin/openedx/core/djangoapps/user_api/accounts/serializers.py

```diff
@@ -0,0 +1,154 @@
+import logging
+
+from django.core.exceptions import ObjectDoesNotExist
+from django.urls import reverse
+
+from common.djangoapps.student.models import PendingNameChange
+from lms.djangoapps.badges.utils import badges_enabled
+from openedx.core.djangoapps.user_api.accounts.utils import is_secondary_email_feature_enabled
+from openedx.core.djangoapps.user_api.accounts.serializers import (
+    AccountLegacyProfileSerializer,
+    LanguageProficiencySerializer,
+    SocialLinkSerializer,
+    get_profile_visibility,
+    get_extended_profile,
+    _visible_fields,
+)
+from openedx.features.name_affirmation_api.utils import get_name_affirmation_service
+
+PROFILE_IMAGE_KEY_PREFIX = 'image_url'
+LOGGER = logging.getLogger(__name__)
+
+
+def to_representation(original_func, self, user):  # lint-amnesty, pylint: disable=arguments-differ
```

Comment:
can we call the original function and modify the result instead of duplicating the whole piece of code?

Reply:
I planned to do this, but this override does not allow it to be implemented this way.